src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Model/ConfigurationSectionSpec.cs

```diff
@@ -10,5 +10,7 @@ internal sealed record ConfigurationSectionSpec : TypeSpec
         public ConfigurationSectionSpec(ITypeSymbol type) : base(type) { }
 
         public override TypeSpecKind SpecKind => TypeSpecKind.IConfigurationSection;
```

Comment:
Do we know if virtual properties are being taken into account in the compiler-generated equality? Assuming they are and assuming it's not desirable these can always be changed to methods.